xbmc/FileItem.cpp

```diff
@@ -3578,11 +3578,56 @@ std::string CFileItem::GetBaseMoviePath(bool bUseFolderNames) const
 
       URIUtils::GetParentPath(name2, strMovieName);
     }
+
+    // Remove 'Disc' subdirectory so actual title passed to scraper
+    if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+            CSettings::SETTING_MYVIDEOS_HANDLEDISCN))
+    {
+      const size_t discPos = GetDiscPath(strMovieName);
+      if (discPos)
+        strMovieName.erase(discPos + 1);
+    }
   }
 
   return strMovieName;
 }
 
+size_t CFileItem::GetDiscPath(const std::string& moviePath) const
+{
+  // Handle Disc, Disk and locale specific spellings
```

Comment:
You need to edit the localizable strings you added to strings.po to properly reflect the changed implementation. Those are talking about 'Disc' only.